lib/interfaces/bdd.js

```diff
@@ -91,6 +91,20 @@ module.exports = function(suite){
     };
 
     /**
+     * Do not randomize.
+     */
+    context.describe.noRandom = function func(title, fn) {
```

Comment:
we could rename this to `sequential`.  and the function as well, not just the object property